symfony/twig-bundle/3.3/config/packages/twig.yaml

```diff
@@ -1,4 +1,6 @@
 twig:
-    paths: ['%kernel.project_dir%/templates']
+    default_path: '%kernel.project_dir%/templates'
+    # additional template paths
+    #paths: []
```

Comment:
@javiereguiluz can you hint when this is needed? Still feels like something we could clean :)
Any other opinion from anyone else?

Reply:
In real apps I've used it 1) to add another directory where templates were defined (yes, I know, this is the obvious purpose!), 2) on symfony.com app I use it to define Twig namespaces, 3) in some personal apps I use it to add `public/static/images/` as a directory for templates. Why? Because that way I can use the SVG icons as regular images in SCSS and embed them in Twig templates with  `{{ source('icons/-------.svg') }}`

But again, if most people never change this option, let's remove it from the default config file. Thanks!

Reply:
Going back to the simpleness version, anyway it's well documented.